hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ConfigurationService.java

```diff
@@ -389,4 +393,19 @@ default UUID updateAsync() {
      * @return registered WAN replication configurations keyed by configuration name
      */
     Map<String, WanReplicationConfig> getWanReplicationConfigs();
+
+    /**
+     * Finds existing namespace configuration by name.
+     *
+     * @param name name of the configuration
+     * @return Namespace configuration or {@code null} when the requested configuration does not exist
+     */
+    NamespaceConfig findNamespaceConfig(String name);
```

Comment:
`findNamespaceConfig` and `getNamespaceConfigs` below seem unused in the project. They should be removed. In particular, `findNamespaceConfig` is irrelevant for namespace configs since we don't apply the lookup-by-pattern config matching style that we apply for data structure configuration.

Reply:
`getNamespaceConfigs` makes sense imo since we're supporting NamespaceConfigs in dynamic configs - agreed on the `findNamespaceConfig` method though 👍

Reply:
Addressed in https://github.com/vbekiaris/hazelcast/commit/cafc0048ba946af66a094d8832bc1dcfb59eed5d